src/steps/campaign-member/campaign-member-field-equals.ts

```diff
@@ -9,7 +9,7 @@ export class CampaignMemberFieldEquals extends BaseStep implements StepInterface
 
   protected stepName: string = 'Check a field on a Salesforce Campaign Member';
   /* tslint:disable-next-line:max-line-length */
-  protected stepExpression: string = 'the salesforce lead (?<email>.+) should have campaign member (?<field>[a-z0-9_]+) (?<operator>set to|not set to|containing|not containing|less than|greater than) (?<expectedValue>.+) on campaign (?<campaignId>.+)';
```

Comment:
- [ ] Campaign Member uses a slightly altered grammar, so be sure to use the appropriate operators / and map them to the expected operators.